src/Manifest.php

```diff
@@ -120,6 +120,17 @@ public function preloadFonts(): void
         ];
     }
 
+    /**
+     * Register MIME types for preloading common web font formats.
```

Comment:
"Register MIME types for preloading stylesheets."

Reply:
fixed